### DISCUSSION TITLE
docs.mockadillo.com

### DISCUSSION THREAD

**Discussion by lnenad** [thuliteio/doks#93] [Showcase]
![mockadillo](https://i.imgur.com/5y3Napi.png)

API Mocking service https://mockadillo.com. We enable you to remove all roadblocks with multiteam development by mocking your entire API infrastructure.

**Comment by h-enk** [comment]
Hello @lnenad, thanks for sharing — nice!

Are your OK with me adding the Mockadillo Documentation website to the Doks showcase?

**Reply by lnenad** [reply]
Yes, of course, that is why I posted here 😄

**Reply by h-enk** [reply]
There it is: https://getdoks.org/showcase/mockadillo-documentation/

**Reply by lnenad** [reply]
Awesome

**Comment by h-enk** [comment]
@lnenad, 

What did you find difficult/what did you struggle with the most when setting up the Mockadillo Documentation website?

**Reply by lnenad** [reply]
Imho it was pretty easy, but there is a lot of custom code, not configurable with toml, that is something that you might want to work on. Also I am using a subdomain config so having `docs.stuff/docs` is a little weird, and it needs to be like that since you've coupled the `/docs` to the theme. All in all thanks for creating a great theme.

**Reply by h-enk** [reply]
Thanks for the feedback — noted.

**Comment by abraviee** [comment]
tested
